Include/cpython/pystate.h

```diff
@@ -148,6 +148,7 @@ struct _ts {
     uint64_t id;
 
     CFrame root_cframe;
+    int use_tracing;
```

Comment:
Shouldn't we add a comment explaining this field only exists for backwards compatibility and is only updated just before calling a trace function? (And in a few other odd places, but the key is that it's not the source of truth any more.)